framework/src/Volo.Abp.GlobalFeatures/Volo/Abp/GlobalFeatures/AbpGlobalFeatureNotEnableException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Volo.Abp.ExceptionHandling;
+
+namespace Volo.Abp.GlobalFeatures
+{
+    public class AbpGlobalFeatureNotEnableException : AbpException, IHasErrorCode
```

Comment:
```suggestion
    public class AbpGlobalFeatureNotEnabledException : AbpException, IHasErrorCode
```

Reply:
Also, please make the exception serializable (using constructor like the AbpException class)

Reply:
Also, please make the exception serializable (using constructor like the AbpException class)